openedx/features/caliper_tracking/tests/current/current_course_enrollment_deactivated.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Rename the branch to 

`tehreem/edx.course.enrollment.deactivated`

Reply:
Also rename the files accordingly. Don't use underscores in filenames or branch names.

The convention is 
simply event name for file name
` edx.course.enrollment.deactivated`

and username/event.name for branches

 `tehreem/edx.course.enrollment.deactivated`